src/components/30-organisms/footer/demo.js

```diff
@@ -62,5 +62,10 @@ storiesOf('Components|Footer/Demos', module)
       span.innerText = `Last clicked link: ${ev.detail}`;
     });
 
+    return wrapper;
+  })
+  .add('Without content', () => {
+    const wrapper = document.createElement('div');
+    wrapper.innerHTML = `<axa-text>This is an emtpy footer. Just for testing.</axa-text><axa-footer></axa-footer>`;
```

Comment:
not very eloquent in my opinion. I would add a footer like this but with the header `AXA WELTWEIT` in addition (only the header):
<img width="1026" alt="Screenshot 2020-07-23 at 09 46 14" src="https://user-images.githubusercontent.com/38284563/88262937-6f898e00-ccc9-11ea-9abd-a152978993f1.png">

And then write something like: No carousel should be rendered on mobile for `AXA WELTWEIT`

Reply:
This is not a common use case. Who wants to have a column with a header but without content? See the issue for that PR. The user do not use column 2. No header, no items.

Reply:
Then don't add the header. I just don't like that there is just a blue square in the demo. You can't tell what the example is about even with a description. It looks broken. In my opinion there should be either the `AXA & SIE` column or the `IN KONTAKT BLEIBEN` column.

Reply:
Ah sorry. Misunderstand that. I want to test that no emtpy accordion is rendered even for column 0. So both columns needs to be emtpy. But i could add the "in kontakt bleiben" column if that is better?